submissions/Zakkarat/js-dom/stylesheet/main.css

```diff
@@ -0,0 +1,229 @@
+body {
+  width: 100%;
+  height: 100%;
+  background-color: #540a24;
+  margin: 0;
+}
+html {
+  height: 100%;
+  overflow-y: hidden;
+}
+a {
+  text-decoration: none;
+  color: white;
+}
+
+/*        HEADER       */
+
+header {
+  width: 100%;
+  min-height: 70px;
+  background-color: #94152d;
+  display: flex;
+
+}
+.toLeft {
+  display: block;
+}
+
+.headerText {
+  justify-content: center;
+  align-items: center;
+  width: 88%;
+  text-align: center;
+  margin: 0;
+  margin-right: 10px;
+  color: white;
+  font-family: Roboto, sans-serif;
+}
+
+/*         NAVBAR          */
+
+.start {
+  left: -280px;
+}
+
+.container {
+  width: 20%;
+  min-width: 280px;
+  height: 92.8%;
+  background-color: #94152d;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  position: fixed;
+  text-align: center;
+  color: white;
+  transition: all 3s;
+}
+.sizeHead {
+  flex-direction: row;
+  justify-content: space-around;
+  visibility: hidden;
+  position: absolute;
+}
+
+
+/*           MAIN             */
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+h1 {
+  color: white;
+  font-family: Roboto;
+  margin-bottom: 2px;
+}
+
+p {
+  font-size: 0.8em;
+  font-family: Roboto;
+  color: white;
+  margin-right: 2%;
+  margin-left: 2%;
+}
+
+img {
+  width: 90%;
+  min-width: 200px;
+  height: 40%;
+  min-height: 185px;
+  border-radius: 5%;
+  transition: 0.6s;
+}
+nav {
+  justify-content: center;
+  font-family: Roboto, sans-serif;
+  font-size: 1.8em;
+}
+
+
+/*           Jhin style            */
+
+.jhin-body {
+  background-color: #540a24;
+  transition: 1s;
+}
+
+.jhin-header {
+  background-color: #94152d !important;
+  transition: 1s;
+}
+.jhin-desk-head {
+  background-color: #821d30 !important;
```

Comment:
This is very bad practice use !important attribute in your css. Can you explain why do you use it in so many cases? Actually you have some pattern in your jhin, cait, kai'sa and MF styles. And you can simplified your css with using classes and toggle them in you js-file. Can you think about it?  If you would stuck with this, write to me here or in the telegram and I'll give you some hints.

Reply:
Done some improvements with `!important` part of code.
Sorry, but I did not understand the part with patterns, I've already have some functions in my app.js, which manipulate classes or did you mean something else? Something like having 3 main classes to store data and change their attributes according to function?

Reply:
It's ok. If we look in your css and js code we can see a lot of stuff with very similar attributes and similar behaviour. For example all your 'hero' classes have propery `transition` and all your js function add text to DOM-elements. We can move most of it to one class/function, and the parts that are different we can move to variable/classes and operate them with js. 
So for example. in your css you will have class `.hero` and than classes `.hero-blue, .hero-red`. than in your js you will have only ONE function that will operate with dom-element with class `.hero` something like that if(clicked on blue){element.toggle('.hero-blue')}. you can move your text into the array method and add appropriate text in you function from this array. Try to think in this way. Don't rack your brain to long, if you will stuck more than couple hours write me here or in the telegram.

Reply:
Done whole thing, but with a little difference - used `style` attribute in HTML, but now it works with only one function. Is it appropriate to use `style` attribute or should I find other ways?

Reply:
Actually this is not very good use styles in your html. But why you don't want to put them into your CSS. I was talking about one function instead of all function in you js, but not instead your functions AND your CSS. you can keep all styles that you need int your CSS. even move colors there for example create classes like .blue .green etc...